backend/core/management/commands/clear_dev_images.py

```diff
@@ -0,0 +1,60 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+"""
+Clear all images that are uploaded when in development mode.
+
+Notes
+-----
+This script is ran in docker-compose.yaml when the backend is started up.
+"""
+
+# ATTN: This file should not be edited as it removes files from a specific local directory.
+
+import os
+
+IMAGE_DIR = os.path.join(
```

Comment:
In `backend/core/settings.py` we already have an environment var called `MEDIA_ROOT`. We should use this var instead, and can remove the code in lines 14-20.

I think the correct usage is `settings.MEDIA_ROOT`?